Demo/Application/Features/ShopperInsightsViewController.swift

```diff
@@ -111,7 +111,15 @@ class ShopperInsightsViewController: PaymentButtonBaseViewController {
             ]
             """
         let paymentMethods = ["Apple Pay", "Card", "PayPal"]
-        shopperInsightsClient.sendPayPalPresentedEvent(paymentMethodsDisplayed: paymentMethods, experiment: sampleExperiment)
+        let presentmentDetails = BTPresentmentDetails(
+            buttonOrder: .first,
+            experimentType: .control,
+            pageType: .about
+        )
+
+//        shopperInsightsClient.sendPayPalPresentedEvent(paymentMethodsDisplayed: paymentMethods, experiment: sampleExperiment)
```

Comment:
We should also be able to remove the `sampleExperiment` and `paymentMethods` properties